DESCRIPTION

```diff
@@ -41,3 +41,4 @@ VignetteBuilder:
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
+Additional_repositories: https://p3m.dev/cran/2024-02-02/
```

Comment:
FYI: CRAN acknowledges this field but doesn't use it AFAIK. `install.packages()` does not use it. `remotes::install_*()` *does* use it, but `pak::pak()` apparently does not (yet) 🤦

Reply:
thanks for the clarification